packages/flutter_test/test/goldens_test.dart

```diff
@@ -148,6 +148,25 @@ void main() {
       });
 
       group('fails', () {
+
+        test('and generates correct output in the correct location', () async {
+          await fs.file(fix('/golden.png')).writeAsBytes(_kColorFailurePngBytes);
+          try {
+            await doComparison();
+            fail('TestFailure expected but not thrown.');
```

Comment:
You might be able to refactor this to use `expect(..., throwsA(TestFailure))` instead, which is usually slightly less error prone than this pattern.

Reply:
(But looks like we are using this pattern elsewhere in this file as well, so feel free to just ignore my comment.)